packages/terra-search-field/src/SearchField.jsx

```diff
@@ -60,6 +60,11 @@ const propTypes = {
    */
   onChange: PropTypes.func,
 
+  /**
+   * Function to trigger when user inputs an value.
```

Comment:
```suggestion
   * Function to trigger when user inputs a value.
```

Reply:
Can we add info on the callback params consumers should expect?

Reply:
Fixed https://github.com/cerner/terra-core/pull/2800/commits/7f1b23a3ff00608c4dcf47ceae24a47d2b94db4d#diff-105fddc0e7a42eb008cb5c09faad5222R64